ios/Classes/RingerModeDetector.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import AudioToolbox
+
+public class RingerModeDetector: NSObject {
+
+    public static let instance = RingerModeDetector()
+
+    var soundId : SystemSoundID = 0
+
+    private static var soundUrl: URL {
+        guard let soundUrl = Bundle(for: RingerModeDetector.self).url(forResource: "silence", withExtension: "aiff") else {
```

Comment:
Where can I find the "silence.aiff"? Is it a default audio file or the phone records a silent audio clip?